extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -148,7 +147,6 @@ tags: {  # <amp-video> in amp-story-page-attachment (same rules as regular AMPHT
     supported_layouts: FIXED
     supported_layouts: FIXED_HEIGHT
     supported_layouts: FLEX_ITEM
-    supported_layouts: INTRINSIC
```

Comment:
@honeybadgerdontcare This also needs to be restored, yeah?

h/t @pierlon https://github.com/ampproject/amp-wp/pull/5703/files#r553119941

Reply:
@westonruter I'm not sure. I went through your last request and thought we had restored what needed to be. If you think `intrinsic` is still missing in areas of this file can you send the PR? You're more familiar with it than I am.

Reply:
@honeybadgerdontcare I'm unsure as I didn't add this `amp-story >> amp-story-page-attachment >> amp-video` instance originally. It was added in e400a655258d614a6248fcb259aee2a11e70bf9f as part of #31073 by @MichaelRybak.

Reply:
The actual change was introduced by #23873 by @gmajoulet (my #31073 was a revision bump).

@gmajoulet, thoughts?

Reply:
Are you trying to know if it's safe to drop support for the INTRINSIC layout for amp-story >> amp-story-page-attachment >> amp-video?

Reply:
@gmajoulet  Yes.

Reply:
@MichaelRybak The change in #23873 by @gmajoulet was to add `amp-story >> amp-story-page-attachment >> amp-video` overall. In #31073 you specifically added the `supported_layouts: INTRINSIC` via aee2260aa4e0dae1b780d7a028d9dc895e44878b. That wasn't intentional?

Reply:
Linking to various PRs isn't helping here. Lets determine what the current state should be and move forward to that.

Reply:
OK. Just trying to capture the history of changes.

Reply:
Hm, I think there was an actual problem with my PR #31073.

TL; DR: It should not have added `supported_layouts: INTRINSIC` into `amp-story >> amp-story-page-attachment >> amp-video`. I think the current state of code in Google and Github is correct.

Here's what happened. The intention was to do a revision bump, i.e. to update validator engine+rules revision after pulling #28349 into Google codebase. Revision bumps are done semi automatically. Our infrastructure creates a CL to pull in the PR, and the CL also updates the revision number. Then the infra creates a PR from that CL to merge the revision back into github - along with merging back the actual code changes - which already exist in github, so the patch should be idempotent/ignored. This last bit (merging) went wrong - instead of recognizing that `supported_layouts: INTRINSIC` is already in `<amp-video> not in amp-story.` section, github applied the patch again, but to a different section, which I didn't notice.

Bottomline, the change was never intended to be applied to `amp-story-page-attachment`. I accidentally added it, and it was later removed both in Google code and on Github. Right now the code is correct.

Reply:
Let's have `amp-story-attachment >> amp-video` strictly follow `amp-video`. Thank you!!

Reply:
Addressed with  CL/351398111 in Google code. The change will be synced to github within a week.

Reply:
The change is now both in Google code (CL/351398111) and in Github (#31922)